docs/docs/main/datasets/uniprot.md

```
@@ -22,11 +22,11 @@ randomly chosen UniRef90 sequence from each.
 ## Data Availability
 
 Two versions of the dataset are distributed, a full training dataset (~80GB) and a 10,000 UniRef50 cluster random slice
-(~150MB). To load and use the sanity dataset, use the [bionemo.core.data.load][bionemo.core.data.load.load] function
+(~150MB). To load and use the sanity dataset, use the [bionemo.common.data.load][bionemo.common.data.load.load] function
```

markdownlint-cli2 (0.22.1)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "bionemo.common.data.load.load"

(MD052, reference-links-images)